projects/demo/src/modules/experimental/card/examples/6/index.less

```diff
@@ -0,0 +1,27 @@
+@import 'taiga-ui-local';
+
+:host {
+    display: flex;
+    gap: 1rem;
+}
+
+.fade {
+    .scrollbar-hidden();
+    width: 100%;
+    height: 2rem;
+    white-space: nowrap;
+    overflow: auto;
+}
+
+.fade-vertical {
+    .scrollbar-hidden();
+    height: 6rem;
+    overflow: auto;
+}
+
+.nowrap {
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    margin: 0;
```

Comment:
Can it just be `div` instead of `p` so we do not need to remove margin?

Reply:
fixed